apps/ops/serializers/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
 #
-
 from .celery import *
+from .variable import *
 from .adhoc import *
```

Comment:
The provided code consists of several imports that seem to be related with different packages (like `celery`, which might refer to a library called Celery). However, it's missing some key sections like header comments, docstrings, etc., making it difficult to analyze its actual function accurately.

Moreover, if you're asking about the difference between two versions (e.g. version 1 vs version 2) instead of the current one (version 3), could you please specify when/how these codes were generated? The purpose behind this inquiry may vary based on context.